### DISCUSSION TITLE
Greetings

### DISCUSSION THREAD

**Discussion by fighting45** [Cyfrin/foundry-full-course-cu#2366] [General]
hi! I made it across the Blockchain basics. Moving on to the solidity development. I have already gone through some basic topic of JavaScript, hopefully that will help.

**Comment by EngrPips** [comment]
Welcome aboard @fighting45,  We wish you the best in your journey.

**Comment by JusticeSol** [comment]
Big ups mate! @fighting45

**Reply by gibhub01** [reply]
welcome same ,just starting out @fighting45  lets connect together and help each other  on x @gibhub01

**Reply by fighting45** [reply]
> welcome same ,just starting out @fighting45  lets connect together and help each other  on x @gibhub01 

Thanks, ill send you a dm. Lets grow together.

**Reply by JusticeSol** [reply]
I just gave you a follow on X @gibhub01 you can follow back

@fighting45 you can follow me on X as well @affanyjoe
Let's connect
